python/nano/test/pytorch/tests/test_inference_pipeline_ipex.py

```diff
@@ -259,3 +271,34 @@ def test_get_model_with_method_name(self):
             assert isinstance(model, PytorchIPEXJITModel)
         except:
             pass
+
+    def test_multiple_input_dataloader(self):
+        # will not run this test if torch < 1.10
+        if TORCH_VERSION_LESS_1_10:
```

Comment:
which skip for this version？

Reply:
for 1.9 ipex, some strange issue might be raised here, since we will gradually close the support for 1.9 so I just skip it here :)